docs/guide/tips-and-tricks.md

```diff
@@ -69,10 +79,10 @@ lines. YAML supports a [variety of options](https://yaml-multiline.info/) to do
 ENCRYPTED values:
 
 ```yaml
-  env:
-    GOOGLE_APPLICATION_CREDENTIALS_DATA: "ENCRYPTED\
-      [3287dbace8346dfbe98347d1954eca923487fd8ea7251983\
-      cb6d5edabdf6fe5abd711238764cbd6efbde6236abd6f274]"
+env:
+  GOOGLE_APPLICATION_CREDENTIALS_DATA: "ENCRYPTED\
+    [3287dbace8346dfbe98347d1954eca923487fd8ea7251983\
+    cb6d5edabdf6fe5abd711238764cbd6efbde6236abd6f274]"
```

Comment:
Could you please revert white space changes?

Reply:
done. sorry about that.

BTW I see that here `env` is used, instead of `environment`. Are they interchangeable?

Reply:
Yeah, `env` and `environment` is the same for the parser.